r/rsparkling/tests/test_as_h2o_frame_df.R

```diff
@@ -0,0 +1,23 @@
+library(testthat)
+library(rsparkling)
+library(dplyr)
```

Comment:
Here and in the other tests we could also load `sparklyr` as we use `spark_connect()` from `sparklyr`. Now it rely on that fact that `rsparkling depends on sparklyr, avoiding depends in favour of imports is a good practice and prevents various issues for users/downstream packages.